### PR TITLE
cloud: refine the kafka changefeed docs

### PR DESCRIPTION
**Pull request by leoppro**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 1]
```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
...
+
+3. If you select **Avro** as your data format, you will see some Avro specific configurations on the page, and you can fill in these configurations as follows:
+
+    - In the **Handle Decimal** and **Handle Bigint Unsigned** configurations, specify how TiDB Cloud handles the decimal and Bigint data types in Kafka messages.
+    - In the **Schema Registry** area, fill in your schema registry endpoints. If you enable **HTTP Authentication**, you also need to fill in the username and password for the schema registry.
+
+4. In the **Topic Distribution** area, select a distribution mode, specify a default Kafka topic name, and set the corresponding topic prefix, separator, and suffix.
+
+    The distribution mode controls how the changefeed creates Kafka topics, by tables, by databases, or cerating one topic for all changelogs. If you select **Avro** as your data format, you can only choose the **Distribute changelogs by table to Kafka Topics** mode in the **Distribute Mode** drop-down list.
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:86]:
```suggestion
    The distribution mode controls how the changefeed creates Kafka topics, by tables, by databases, or creating one topic for all changelogs. If you select **Avro** as your data format, you can only choose the **Distribute changelogs by table to Kafka Topics** mode in the **Distribute Mode** drop-down list.
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 2]
```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
 
 If your Apache Kafka service is in a GCP VPC that has no internet access, take the following steps:
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:32]:
```suggestion
If your Apache Kafka service is in a GCP VPC that has no Internet access, take the following steps:
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+Make sure that your TiDB cluster can connect to the Apache Kafka service.
 
 If your Apache Kafka service is in an AWS VPC that has no internet access, take the following steps:
 
-1. [Set up a VPC peering connection](/tidb-cloud/set-up-vpc-peering-connections.md) between the VPC of the Apache Kafka service and your TiDB cluster. 
-2. Modify the inbound rules of the security group that the Apache Kafka service is associated with. 
+1. [Set up a VPC peering connection](/tidb-cloud/set-up-vpc-peering-connections.md) between the VPC of the Apache Kafka service and your TiDB cluster.
+2. Modify the inbound rules of the security group that the Apache Kafka service is associated with.
 
     You must add the CIDR of the region where your TiDB Cloud cluster is located to the inbound rules. The CIDR can be found on the VPC Peering page. Doing so allows the traffic to flow from your TiDB cluster to the Kafka brokers.
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:25]:
```suggestion
    You must add the CIDR of the region where your TiDB Cloud cluster is located to the inbound rules. The CIDR can be found on the **VPC Peering** page. Doing so allows the traffic to flow from your TiDB cluster to the Kafka brokers.
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 4]
```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
...
+
+1. In the TiDB Cloud console, navigate to the **Clusters** page for your project.
+2. Click the name of the cluster that you want to create a changefeed for.
+3. Click the **Changefeed** tab.
+4. Click **Sink to Apache Kafka**.
+
+## Step 2. Configure the changefeed target
+
+1. Under **Brokers Configuration**, fill in your Kafka brokers endpoints. You can use commas `,` to separate multiple endpoints.
+2. Select your Kafka version. If you do not known that, use Kafka V2.
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:58]:
```suggestion
2. Select your Kafka version. If you do not known that, use **Kafka V2**.
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 5]
```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
...
+3. Select a desired compression type for the stream data.
+4. Enable the **TLS Encryption** option if your Kafka have enabled TLS encryption and you want to use TLS encryption for the Kafka connection.
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:60]:
```suggestion
4. Enable the **TLS Encryption** option if your Kafka has enabled TLS encryption and you want to use TLS encryption for the Kafka connection.
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 6]
```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
...
+    - If your Kafka does not require authentication, keep the default option **Disable**.
+    - If your Kafka requires authentication, select the corresponding authentication type, and then fill in the user name and password of your Kafka account for authentication.
+
+6. Click **Next** to check the configurations you set and go to the next page.
+
+## Step 3. Set the changefeed
+
+1. In the **Data Format** area, select your desired format of Kafka messages.
+
+   - Avro is a compact, fast, binary data format with rich data structures, which is widely used in various flow systems. For more information, see [Avro data format](https://docs.pingcap.com/tidb/stable/ticdc-avro-protocol).
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:72]:
```suggestion
   - Avro is a compact, fast, and binary data format with rich data structures, which is widely used in various flow systems. For more information, see [Avro data format](https://docs.pingcap.com/tidb/stable/ticdc-avro-protocol).
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 7]
```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
...
+   - Canal-JSON is a plain JSON text format, which is easy to parse. For more information, see [Canal-JSON data format](https://docs.pingcap.com/tidb/stable/ticdc-canal-json).
+
+2. Enable the **TiDB Extension** option if you want to add TiDB-extension fields to the Kafka message body.
+
+    For more information about TiDB-extension fields, see [TiDB extension fields in Avro data format](https://docs.pingcap.com/tidb/stable/ticdc-avro-protocol#tidb-extension-fields) and [TiDB extension fields in Canal-JSON data format](https://docs.pingcap.com/tidb/stable/ticdc-canal-json#tidb-extension-field).
+
+3. If you select **Avro** as your data format, you will see some Avro specific configurations on the page, and you can fill in these configurations as follows:
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:79]:
```suggestion
3. If you select **Avro** as your data format, you will see some Avro-specific configurations on the page. You can fill in these configurations as follows:
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 8]
```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
...
+
+    - In the **Handle Decimal** and **Handle Bigint Unsigned** configurations, specify how TiDB Cloud handles the decimal and Bigint data types in Kafka messages.
+    - In the **Schema Registry** area, fill in your schema registry endpoints. If you enable **HTTP Authentication**, you also need to fill in the username and password for the schema registry.
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:82]:
```suggestion
    - In the **Schema Registry** area, fill in your schema registry endpoints. If you enable **HTTP Authentication**, you also need to fill in the user name and password for the schema registry.
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 9]
```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
...
+After a changefeed is created, you can navigate to the **Changefeed** tab of your TiDB cluster and click **Sink to Apache Kafka** to open the **Changefeed Detail** dialog.
 
-## Delete a Sink
+In the **Changefeed Detail** dialog, you can manage the changefeed as follows:
 
-1. Navigate to the **Changefeed** tab of a cluster.
-2. Click the trash button of **Sink to Apache Kafka**
+- Check the running state of the changefeed.
+- Click **Delete** to delete the changefeed.
+- Click **Pause** or **Resume** to pause or resume the changefeed.
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:133]:
```suggestion
- Check the running state of the changefeed.
- Delete the changefeed by clicking **Delete**.
- Pause or resume the changefeed by clicking **Pause** or **Resume**.
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 10]
```diff
@@ -1,26 +1,26 @@
...
+> For [Developer Tier clusters](/tidb-cloud/select-cluster-tier.md#developer-tier), the changefeed feature is unavailable.
 
 ## Prerequisites
 
 ### Network
 
-Make sure that your TiDB Cluster can connect to the Apache Kafka service.
+Make sure that your TiDB cluster can connect to the Apache Kafka service.
 
 If your Apache Kafka service is in an AWS VPC that has no internet access, take the following steps:
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:20]:
```suggestion
If your Apache Kafka service is in an AWS VPC that has no Internet access, take the following steps:
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 11]
```diff
@@ -31,29 +31,106 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
...
+## Step 4. Review the configurations
 
-You must prepare a Topic before creating an Apache Kafka Sink. Based on table, the Sink will distribute data to different partitions of the Topic.
+On this page, you can review all the changefeed configurations that you set.
 
-## Create a Sink
+If you find any error, you can go back to fix the error. If there is no error, you can click the check box at the bottom, and then click **Create** to create the changefeed.
 
-After completing the Prerequisites, you can sink your data to Apache Kafka.
+## Step 5. Manage the changefeed
```
Comment (hfxsd) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:125]: This section includes changefeed operations, not a necessary step of the whole procedure. Better make it an independent section. 
```suggestion
## Manage the changefeed
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: tidb-cloud/changefeed-sink-to-apache-kafka.md (hunk 2 above)
Comment (qiancai) [reply tidb-cloud/changefeed-sink-to-apache-kafka.md:32]: According to [Capitalization of Internet - Wikipedia](https://en.wikipedia.org/wiki/Capitalization_of_Internet), the spelling internet has become often used, as the word almost always refers to the global network; the generic sense of the word has become rare in non-technical writings. As a result, various [style manuals](https://en.wikipedia.org/wiki/Style_guide), including [The Chicago Manual of Style](https://en.wikipedia.org/wiki/The_Chicago_Manual_of_Style), the [Associated Press](https://en.wikipedia.org/wiki/Associated_Press)'s [AP Stylebook](https://en.wikipedia.org/wiki/AP_Stylebook), and the [AMA Manual of Style](https://en.wikipedia.org/wiki/AMA_Manual_of_Style), revised their formerly capitalized stylization of the word to lowercase internet in 2016.

File: tidb-cloud/changefeed-sink-to-apache-kafka.md (hunk 4 above)
Comment (qiancai) [reply tidb-cloud/changefeed-sink-to-apache-kafka.md:58]: The actual version provided on the UI is quite long. In this case, do we need to use bold text for Kafka V2?

image

Comment (qiancai) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:58]:
```suggestion
2. Select your Kafka version. If you do not know that, use Kafka V2.
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 12]
```diff
@@ -31,30 +31,109 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
...
-1. Navigate to the **Changefeed** tab of a cluster.
-2. Click the trash button of **Sink to Apache Kafka**
+- Check the running state of the changefeed.
+- Delete the changefeed by clicking **Delete**.
+- Pause or resume the changefeed by clicking **Pause** or **Resume**.
 
 ## Restrictions
 
-Because TiDB Cloud uses TiCDC to establish changefeeds, it has the same [restrictions as TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview#restrictions).
+- Only one kafka changefeed of a TiDB cluster can be created.
```
Comment (qiancai) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:137]:
```suggestion
- For each TiDB Cloud cluster, you can create only one Kafka changefeed.
```

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 13]
```diff
@@ -31,30 +31,109 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
...
+- Do not support uploading self-signed TLS certificates to connect to Kafka brokers temporarily.
```
Comment (qiancai) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:138]:
```suggestion
- Currently, TiDB Cloud does not support uploading self-signed TLS certificates to connect to Kafka brokers.
```

**Review by cchenkey — APPROVED** [review]

File: tidb-cloud/changefeed-sink-to-apache-kafka.md [hunk 14]
```diff
@@ -31,30 +31,109 @@ If your Apache Kafka service is in an AWS VPC that has no internet access, take
...
+1. Under **Brokers Configuration**, fill in your Kafka brokers endpoints. You can use commas `,` to separate multiple endpoints.
+2. Select your Kafka version. If you do not know that, use Kafka V2.
+3. Select a desired compression type for the stream data.
+4. Enable the **TLS Encryption** option if your Kafka has enabled TLS encryption and you want to use TLS encryption for the Kafka connection.
+5. Select the **Authentication** option according to your Kafka authentication configuration.
+
+    - If your Kafka does not require authentication, keep the default option **Disable**.
+    - If your Kafka requires authentication, select the corresponding authentication type, and then fill in the user name and password of your Kafka account for authentication.
+
+6. Click **Next** to check the configurations you set and go to the next page.
```
Comment (qiancai) [inline tidb-cloud/changefeed-sink-to-apache-kafka.md:66]:
```suggestion
3. Enable the **TLS Encryption** option if your Kafka has enabled TLS encryption and you want to use TLS encryption for the Kafka connection.
4. Select the **Authentication** option according to your Kafka authentication configuration.

    - If your Kafka does not require authentication, keep the default option **Disable**.
    - If your Kafka requires authentication, select the corresponding authentication type, and then fill in the user name and password of your Kafka account for authentication.

5. Click **Next** to check the configurations you set and go to the next page.
```

Comment (qiancai) [reply tidb-cloud/changefeed-sink-to-apache-kafka.md:66]: @leoppro PTAL

Comment (leoppro) [reply tidb-cloud/changefeed-sink-to-apache-kafka.md:66]: LGTM

**Review by hfxsd — APPROVED** [review]
LGTM

**Pull request merged**